pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -153,10 +154,13 @@ func (o *EditOptions) Complete(f cmdutil.Factory, out, errOut io.Writer, args []
 }
 
 // Validate checks the EditOptions to see if there is sufficient information to run the command.
+// TODO: add the implement of Validate
```

Comment:
Why not implement it now?

Reply:
Beause  this PR is mainly improving documentatiom and I will send a PR about Validate later